ddl/backfilling.go

```diff
@@ -480,7 +481,8 @@ func (dc *ddlCtx) sendTasksAndWait(scheduler *backfillScheduler, totalAddedCount
 		zap.String("start key", hex.EncodeToString(startKey)),
 		zap.String("next key", hex.EncodeToString(nextKey)),
 		zap.Int64("batch added count", taskAddedCount),
-		zap.String("take time", elapsedTime.String()))
+		zap.String("take time", elapsedTime.String()),
```

Comment:
Since this is a backport, I think it is better to keep it in line with master:
https://github.com/pingcap/tidb/blob/723526708ddcdf63a40b264ce1f63eaec6885001/ddl/backfilling.go#L596